docs/ops/production_ready.md

```diff
@@ -52,18 +52,6 @@ By default, operator uids are generated by traversing the JobGraph and hashing s
 While this is comfortable from a user perspective, it is also very fragile, as changes to the JobGraph (e.g., exchanging an operator) results in new UUIDs.
 To establish a stable mapping, we need stable operator uids provided by the user through `setUid(String uid)`.
 
-### Choose The Right State Backend
-
-Currently, Flink's savepoint binary format is state backend specific.
-A savepoint taken with one state backend cannot be restored using another, and you should carefully consider which backend you use before going to production.
```

Comment:
I know this is not related to this PR but is this actually still true? @carp84? If not, then we should probably update it.

Reply:
Sorry for the late reply. And yes, this is still true. The efforts of unifying savepoint binary format and supporting state backend switch is tracked by [FLIP-41](https://cwiki.apache.org/confluence/display/FLINK/FLIP-41%3A+Unify+Binary+format+for+Keyed+State) but not done yet.

Reply:
Ok, thanks for the confirmation @carp84.